optimize/upgrade/src/test/java/io/camunda/optimize/upgrade/main/UpgradeProcedureTest.java

```diff
@@ -103,25 +103,6 @@ public void initializeSchemaIsNotCalled() {
     verify(schemaUpgradeClient, never()).initializeSchema();
   }
 
-  @Test
-  public void validationIsDoneBeforeUpgradeExecution() {
```

Comment:
Why is this test no longer relevant?

Reply:
I was wrong. This can remain, and will work as soon as the project version is updated on the branch changes that Roman is handling

Reply:
Merged [here](https://github.com/camunda/camunda/pull/27210), so I have rebased this PR